app/src/main/res/layout/list_users_adapter.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Ok, it's true. this layout is used for RecyclerView's Adapter. frameLayout erased.